SR2021/2021-01-08-code-submitter-credentials.md

```diff
@@ -0,0 +1,26 @@
+---
+to: SR2021 teams
+subject: Student Robotics Code Submitter Credentials
+---
+
+Hi,
+
+Here are your credentials for using the [code submitter](https://studentrobotics.org/code-submitter/), the tool we're using to allow you to submit your code to us for running in a competition event.
+
+The code should be in a zip file, and the entry point for your robot program named `robot.py`. Note that anyone in your team can submit code to the submitter, but only the latest upload will be used.
+
+When visiting the code submitter, you'll be prompted for a username and password. The username will be your TLA, and the password is:
```

Comment:
I don't think anyone would try and copy paste a table as a password but maybe better to be explicit?
```suggestion
When visiting the code submitter, you'll be prompted for a username and password. The username will be your TLA, and your password can be found in the table below:
```